media-libs/lib3mf/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>scott@alfter.us</email>
+		<name>Scott Alfter</name>
+	</maintainer>
```

Comment:
Please add someone to commit for you, it's usually the proxy-maint project. See example from other proxy-maintained ebuilds how to correctly do it.